common-scripts/basic-gluster.sh

```diff
@@ -128,3 +128,12 @@ then
 	echo "Export ${GLUSTER_VOLUME} is not available"
 	exit 1
 fi
+
+if [ "${ENABLE_ACL}" == "True" ]
+then
+  conf_file="/etc/ganesha/exports/export."${GLUSTER_VOLUME}".conf"
+  sed -i s/'Disable_ACL = .*'/'Disable_ACL = false;'/g ${conf_file}
+  cat ${conf_file}
+  dbus-send --type=method_call --print-reply --system  --dest=org.ganesha.nfsd /org/ganesha/nfsd/ExportMgr  org.ganesha.nfsd.exportmgr.UpdateExport string:${conf_file} string:"EXPORT(Export_Id = 2)"
```

Comment:
Though the ExportID is always '2' for now, can we grep it from .conf file? In case if it changes in future.